src/validations/request.validation.js

```diff
@@ -12,41 +11,48 @@ import TripService from '../services/trip.service';
 */
 async function requestValidation(req, res, next) {
   const signInUser = JwtService.verifyToken(req.headers.authorization);
```

Comment:
Remove this line and use the designated auth middleware to validate that a user is logged in